test_regress/t/t_wait_notiming.out

```diff
@@ -0,0 +1,14 @@
+%Error-UNSUPPORTED: t/t_wait.v:12:7: Unsupported: wait statements
```

Comment:
Should this be instead

```suggestion
%Error-UNSUPPORTED: t/t_wait.v:12:7: Unsupported: wait statements, as using --no-timing
```

so that users know there's now an alternative?